apps/main/src/dao/types/dao.types.ts

```diff
@@ -1,4 +1,4 @@
-import type { INetworkName } from '@curvefi/api/lib/interfaces'
+import type { IChainId, INetworkName } from '@curvefi/api/lib/interfaces'
```

Comment:
Ser can you please fix your prettier 😭 
The PR reviews get really unwieldy if we keep up like this lol